documentation/quick_start_qemu/quick_start_qemu_linux.md

```diff
@@ -81,7 +127,7 @@ Using the `scons` command to compile the BSP.
 
 ![compile the BSP](figures/ubuntu-scons.png)
 
-## Introduction of QEMU BSP Catalogue
+## 4 Introduction of QEMU BSP Catalogue
 
 The board-level support package (BSP) provided by RT-Thread simulates ARM vexpress A9 development board is located in the `qemu-vexpress-a9` folder under the `bsp` directory of RT-Thread source code. This BSP implements LCD, keyboard, mouse, SD card, Ethernet card, serial port and other related drivers. The contents of the folder are shown in the following figure.
```

Comment:
The board-level support package (BSP) provided by RT-Thread to simulate ARM vexpress A9 development board locates in the `qemu-vexpress-a9` folder under the `bsp` directory of RT-Thread source code. This BSP implements LCD, keyboard, mouse, SD card, Ethernet card, serial port and other related drivers. The contents of the folder are shown in the following figure.